packages/artifacts/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src", "test", "rollup.config.ts"]
+  "include": ["src", "test", "rollup.config.ts"],
+  "compilerOptions": {
+    "baseUrl": "src"
```

Comment:
It's working but I think this option should be copied in the `tsconfig.build.json` file too.